products/compute/api.yaml

```diff
@@ -659,7 +658,85 @@ objects:
           that was later deleted and recreated under the same name, the source
           image ID would identify the exact version of the image that was used.
         output: true
-<%= indent(compile('products/compute/disk_parameters.yaml'), 6) %>
+      - !ruby/object:Api::Type::NestedObject
```

Comment:
Have any lines below here been modified? Or is it all copied directly from the other file? My expectation is that a few `min_version: beta` lines have been removed based on the downstreams, but as-is this diff doesn't make that clear.

Ideally if this is a copy-and-change the copy and change would be broken into separate commits.

Reply:
Ideally, you are correct. But the commit message indicates that both happened, as does the PR title. IMO it's not a change that blocks this PR.